auth-saml-ldap/app/routes/auth/slo.tsx

```diff
@@ -0,0 +1,10 @@
+import type { ActionArgs } from "@remix-run/node";
+
+import { logout } from "~/session.server";
+
+/* can't do idp initiated logout w/ cookie sessions, but can still use
+   this point to logout if we wanna
+*/
+export const action = async ({ request }: ActionArgs) => {
```

Comment:
Args aren't used

```suggestion
export const action = async () => {
```

Reply:
in this one should it be 
```js
export const action = async (request: Request) => {
  return await logout(request);
};
```

since request is used in the logout function, or does it automatically include in action functions?